src/index.ts

```diff
@@ -3,56 +3,82 @@ import { ModuleWithProviders, NgModule } from '@angular/core';
 export { NgAisBreadcrumbModule };
+
 import { NgAisClearRefinementsModule } from './clear-refinements/clear-refinements.module';
 export { NgAisClearRefinementsModule };
+
 import { NgAisCurrentRefinementsModule } from './current-refinements/current-refinements.module';
 export { NgAisCurrentRefinementsModule };
+
 import { NgAisHierarchicalMenuModule } from './hierarchical-menu/hierarchical-menu.module';
 export { NgAisHierarchicalMenuModule };
+
 import { NgAisHitsPerPageModule } from './hits-per-page/hits-per-page.module';
 export { NgAisHitsPerPageModule };
+
 import { NgAisHitsModule } from './hits/hits.module';
 export { NgAisHitsModule };
+
 import { NgAisIndexModule } from './index-widget/index-widget.module';
 export { NgAisIndexModule };
+
 import { NgAisInfiniteHitsModule } from './infinite-hits/infinite-hits.module';
 export { NgAisInfiniteHitsModule };
+
 import { NgAisInstantSearchModule } from './instantsearch/instantsearch.module';
 export { NgAisInstantSearchModule };
+
 import { NgAisMenuModule } from './menu/menu.module';
 export { NgAisMenuModule };
+
 import { NgAisNumericMenuModule } from './numeric-menu/numeric-menu.module';
 export { NgAisNumericMenuModule };
+
 import { NgAisPaginationModule } from './pagination/pagination.module';
 export { NgAisPaginationModule };
+
 import { NgAisRangeSliderModule } from './range-slider/range-slider.module';
 export { NgAisRangeSliderModule };
+
 import { NgAisRefinementListModule } from './refinement-list/refinement-list.module';
 export { NgAisRefinementListModule };
+
 import { NgAisSearchBoxModule } from './search-box/search-box.module';
 export { NgAisSearchBoxModule };
+
 import { NgAisSortByModule } from './sort-by/sort-by.module';
 export { NgAisSortByModule };
+
 import { NgAisRatingMenuModule } from './rating-menu/rating-menu.module';
 export { NgAisRatingMenuModule };
+
 import { NgAisStatsModule } from './stats/stats.module';
 export { NgAisStatsModule };
+
 import { NgAisToggleModule } from './toggle/toggle.module';
 export { NgAisToggleModule };
+
 import { NgAisHighlightModule } from './highlight/highlight.module';
 export { NgAisHighlightModule };
+
 import { NgAisRangeInputModule } from './range-input/range-input.module';
 export { NgAisRangeInputModule };
+
 import { NgAisPanelModule } from './panel/panel.module';
 export { NgAisPanelModule };
+
 import { NgAisConfigureModule } from './configure/configure.module';
 export { NgAisConfigureModule };
+
 import { NgAisConfigureRelatedItemsModule } from './configure-related-items/configure-related-items.module';
 export { NgAisConfigureRelatedItemsModule };
+
 import { NgAisQueryRuleCustomDataModule } from './query-rule-custom-data/query-rule-custom-data.module';
 export { NgAisQueryRuleCustomDataModule };
+
 import { NgAisQueryRuleContextModule } from './query-rule-context/query-rule-context.module';
 export { NgAisQueryRuleContextModule };
+
 import { NgAisVoiceSearchModule } from './voice-search/voice-search.module';
```

Comment:
we can import and export on the same line to make this easier to read:

```suggestion
export { NgAisVoiceSearchModule } from './voice-search/voice-search.module';
```

Reply:
I may be misunderstanding your point. If I change here to `export` I will not be able to reference `NgAisVoiceSearchModule` somewhere else in the file, in the `NGIS_MODULES`

```ts
const NGIS_MODULES = [
  ...
  NgAisVoiceSearchModule,
];
```

Did see this reference?